torchbox/wagtail-torchbox
# Adds a hard-coded external link to ai.torchbox.com

https://torchbox.monday.com/boards/1192293412/views/4019870/pulses/1207882157
Design as discussed on slack with Ben.

### Changes included with this MR
    - adds external link icon
    - adds ability to preview sprites in the pattern library
    - adjusts font-sizing and spacing of main navigation menu to accommodate the extra link
    - adds a new break-point to assist with fitting in the extra link

### To test
- On a local build, visit the home page to see how the new link appears in the navigation and test it at various resolutions, noting how the font-size changes
- Visit each top level proposition page to see how the new link appears in the navigation, and test the hover state
- Check the mobile drop-down menu style for the new link for each top level proposition page

### **Screenshots**

### Home page at 4 different resolutions

![Screenshot 2023-06-12 at 12 58 41](https://github.com/torchbox/wagtail-torchbox/assets/771869/48c4e1ab-12f9-4bea-9d11-6a2228da5fbf)

![Screenshot 2023-06-12 at 12 58 33](https://github.com/torchbox/wagtail-torchbox/assets/771869/41561659-d877-4ac8-9d25-f121cd4a4272)

![Screenshot 2023-06-12 at 12 58 17](https://github.com/torchbox/wagtail-torchbox/assets/771869/599a1978-2651-4e9c-89e6-0f2b6f936a5f)

![Screenshot 2023-06-12 at 12 58 08](https://github.com/torchbox/wagtail-torchbox/assets/771869/f17ff222-f0ef-452b-b5de-7f72008c755f)

### Inner page mobile menu

![Screenshot 2023-06-12 at 13 00 11](https://github.com/torchbox/wagtail-torchbox/assets/771869/252fd953-9ac3-4e31-8481-4f07c2a65434)

### Inner page desktop menu

![Screenshot 2023-06-12 at 13 01 21](https://github.com/torchbox/wagtail-torchbox/assets/771869/dd2e31da-524a-4cba-9ce6-9ca4f173147d)

### Inner page alternative theme

![Screenshot 2023-06-12 at 13 02 34](https://github.com/torchbox/wagtail-torchbox/assets/771869/d2c762d9-4ccf-42dd-a917-559d3c3b5d60)
